spec/policies/agent/rdv_policy_spec.rb

```diff
@@ -136,5 +136,24 @@
     end
   end
 
+  context "any participating user is soft deleted" do
+    let(:organisation) { create(:organisation) }
+    let(:service) { create(:service) }
+    let(:agent) { create(:agent, admin_role_in_organisations: [organisation], service: service) }
+    let(:motif) { create(:motif, organisation: organisation, service: service) }
+
+    let(:live_user)         { create(:user, deleted_at: 2.weeks.ago) }
```

Comment:
je pense que c’est une erreur ici non ?

```suggestion
    let(:live_user)         { create(:user, deleted_at: nil) }
```